packages/connect-web/browserstack/karma-fixup-AbortController.js

```diff
@@ -16,6 +16,16 @@
 
 (function () {
   if (globalThis.AbortController && globalThis.AbortSignal) {
+    // `throwIfAborted` was added much later.
+    if (!globalThis.AbortSignal.prototype.throwIfAborted) {
```

Comment:
Ah, forgot about that when suggesting `throwIfAborted` for the @connectrcp/connect-web transports. I don't think we should start to require for v1. It's very possible that it requires someone to add additional polyfills for old browsers in our next v1 release.

We call it to work around an issue in the Node.js fetch implementation. The `signal` option was added much earlier to browsers than `throwIfAborted` (compare [init.signal](https://developer.mozilla.org/en-US/docs/Web/API/Window/fetch#browser_compatibility) to [throwIfAborted](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/throwIfAborted#browser_compatibility)). I expect browsers to _not_ have the same issue as Node.js.

Following this assumption, we can update the transports to use:

```ts
if ("throwIfAborted" in signal) {
  // We assume that implementations without `throwIfAborted` (old 
  // browsers) do honor aborted signals on `read`.
  signal.throwIfAborted();
}
```

WDYT?